apps/gro/views/no-postcode.html

```diff
@@ -0,0 +1,60 @@
+{{<layout}}
+  {{$step}}Step 2{{/step}}
+
+  {{$journeyHeader}}
+    {{#t}}journey.header{{/t}}
+  {{/journeyHeader}}
+
+  {{$validationSummary}}
+    {{> partials-validation-summary}}
+  {{/validationSummary}}
+
+  {{$propositionHeader}}
+    {{> partials-navigation}}
+  {{/propositionHeader}}
+
+  {{$header}}{{#t}}pages.post.header{{/t}}{{/header}}
+
+  {{$content}}
+
+    {{<partials-form}}
+
+      {{$form}}
+        <p>
+          <span class="postcode">{{values.postcode}}</span>
+          <span class="link"><a href="{{baseUrl}}/postcode">{{#t}}pages.address.edit{{/t}}</a></span>
+        </p>
+
+        <legend class="visuallyhidden">
+          <span class="form-label-bold">{{values.postcode-error}}</span>
+        </legend>
```

Comment:
I fairly certain that if a `legend` is included then it should follow immediately after the opening `fieldset` tag